l10n_ec_credit_note/views/res_config_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field
+            name="inherit_id"
+            ref="l10n_ec_account_edi.res_config_settings_l10n_ec_edi"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='l10n_ec_section']" position="inside">
+                <block title="Credit Note" id="l10n_ec_credit_note">
+                    <setting string="Default Accounts for  Credit Notes">
+                        <div class="content-group">
+                            <div class="row mt8">
+                                <label
+                                    for="l10n_ec_property_account_discount_id"
+                                    class="col-lg-5 o_light_label"
+                                    string="C.C. Discount"
+                                />
+                                <field
+                                    name="l10n_ec_property_account_discount_id"
+                                    placeholder="Same Account"
```

Comment:
What do you mean by "Same Account"? It's better to leave this placeholder empty because it is confusing.

Reply:
ok remove placeholder